README.md

```diff
@@ -15,6 +15,61 @@ def deps do
 end
 ```
 
+## Usage
+
+Validate data against a defined schema.
+
+* `Litmus.validate/2`
```

Comment:
Remove this line. The function will be listed in the docs, I don't think you need to re-list it here.

Reply:
okay